java/openmldb-taskmanager/src/main/scala/com/_4paradigm/openmldb/taskmanager/JobInfoManager.scala

```diff
@@ -149,7 +149,6 @@ object JobInfoManager {
     statement.setString(8, job.getApplicationId)
     statement.setString(9, job.getError)
 
-    var pstmt: PreparedStatement = null
     try {
```

Comment:
looks wired, maybe u can use `try with resource`  alternative in scala: https://www.baeldung.com/scala/try-with-resources